### DISCUSSION TITLE
Dynamic DisplayName

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5322] [Help]
![ilkdosya](https://user-images.githubusercontent.com/24252615/103076618-31c5c400-45df-11eb-9c32-7e4b2fb4a609.png)
![ikincidosya](https://user-images.githubusercontent.com/24252615/103076623-368a7800-45df-11eb-9a75-286a1c725090.png)

I would like to know if it is possible to change the display name of a field Dynamically.
I want to bring the names of the records in the tours table to the turns1 tour2 tour3 columns in the first table. Is it possible and I can do this? Is there an example?

For example I want the display name of the Tour1 column to be yt pospoid, which is the first record of the tours table.

**Comment by VictorTomaili** [comment]
Edit your MagazaHedefColumns.cs file for show foreign name field.   
Example:

[OrderRow.cs#L46](https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Core/Modules/Northwind/Order/OrderRow.cs#L46)
[OrderColumns.cs#L20](https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Core/Modules/Northwind/Order/OrderColumns.cs#L20)

**Comment by slmysl** [comment]
Do you mean that you have two table, one of them is colums name table and the other one is value table?
I mean on magaza hedef you want to write 'YT Postpaid' instead of 'Tur 1' as column name.
am I right?

**Reply by havvanur160401046** [reply]
Yes

**Reply by slmysl** [reply]
I have the same problem. lol

**Reply by VictorTomaili** [reply]
oh, it's too much complex than I thought, you need customize your columns on grid.ts. There is no sample for this in serene. But we done custom something similar to this for our customers already.

**Reply by slmysl** [reply]
Can you please write the ts code ?

**Reply by VictorTomaili** [reply]
Nope it's not free public code. It's feature for our customer I said. You can write your own.

**Reply by slmysl** [reply]
10x :)
